Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -41,11 +40,6 @@ public class ListViewRenderer : ViewRenderer<ListView, UITableView>
 		protected UITableViewRowAnimation InsertRowsAnimation { get; set; } = UITableViewRowAnimation.Automatic;
 		protected UITableViewRowAnimation DeleteRowsAnimation { get; set; } = UITableViewRowAnimation.Automatic;
 		protected UITableViewRowAnimation ReloadRowsAnimation { get; set; } = UITableViewRowAnimation.Automatic;
-		protected UITableViewRowAnimation ReloadSectionsAnimation
```

Comment:
This is not being used anywhere.

Reply:
keep it. it might be there so it can be accessed by subclasses and custom renderers